main.tf

```diff
@@ -0,0 +1,95 @@
+#Internet VPC
+resource "aws_vpc" "main" {
+    cidr_block = "10.0.0.0/16"
+    instance_tenancy = "default"
+    enable_dns_support = "true"
+    enable_dns_hostnames = "true"
+    enable_classiclink = "false"
+    
+    tags = {
+        Name = "main"
+    }
+}
+
+#Subnets
+resource "aws_subnet" "main-public" {
+    count = 2
+    vpc_id = "${aws_vpc.main.id}"
+    cidr_block = "10.0.${count.index+1}.0/24"
+    map_public_ip_on_launch = "true"
+    availability_zone = "${count.index==0 ? "us-east-1a" : "us-east-1b"}"
+    tags = {
+        Name = "main-public-${count.index+1}"
+    }
+}
+
+resource "aws_subnet" "main-private" {
+    count = 2
+    vpc_id = "${aws_vpc.main.id}"
+    cidr_block = "10.0.${count.index+3}.0/24"
+    map_public_ip_on_launch = "false"
+    availability_zone = "${count.index==0 ? "us-east-1a" : "us-east-1b"}"
+    tags = {
+        Name = "main-private-${count.index+1}"
+    }
+}
+
+#Internet GW
+resource "aws_internet_gateway" "main-gw" {
+    vpc_id = "${aws_vpc.main.id}"
+    tags = {
```

Comment:
It's good to separate `tag` from the rest of the arguments using blank line as well

Reply:
fixed